conda-recipes/cyclus-deps/post-link.sh

```diff
@@ -1,30 +1,3 @@
-#/bin/bash
```

Comment:
Maybe this just means I don't understand conda well enough, but why don't we need any of this anymore?